wct.conf.json

```diff
@@ -28,7 +28,7 @@
 				{
 					"browserName": "safari",
 					"platform": "OS X 10.12",
-					"version": "10.0"
+					"version": "11.0"
```

Comment:
This can just be left blank to use the latest version (I think using `"latest"` also works?)

Reply:
Right, I'm going to make this change because it's bit me a few times now.